common/web/keyboard-processor/src/keyboards/activeLayout.ts

```diff
@@ -23,6 +23,24 @@ function Enumerable(
     descriptor.enumerable = true;
 };
 
+const KeyTypesOfKeyMap = {
```

Comment:
Is there no way we can use the type data from common/web/types? This is just a bit WET